src/CarbonAware.DataSources/CarbonAware.DataSources.WattTime/src/WattTimeDataSource.cs

```diff
@@ -25,7 +25,7 @@ public class WattTimeDataSource : ICarbonIntensityDataSource
 
     private IWattTimeClient WattTimeClient { get; }
 
-    private ActivitySource ActivitySource { get; }
+    private static readonly ActivitySource Activity = new ActivitySource(nameof(WattTimeDataSource));
```

Comment:
What was the rationale for starting this here? I have vague memories that @bderusha was very particular about where he initialized the Activity source but I'm not familiar with that reasoning

Reply:
On second thought the important piece may be where it's started, not where it's initialized, so large grain-of-🧂 with this question^^

Reply:
BTW, some of these changes are based on the fact that there was a lot of inconsistencies along the code based related with activities.

Reply:
Ahh nice, love best practices docs 🤓 Thanks for bringing up to consistency